### ISSUE TITLE
near.org Notifications about DevHub posts lead to error message

### ISSUE THREAD

**Issue by ori-near** [near/neardevhub-widgets#504]
# Describe the bug
NEAR Social provides notifications to some DevHub activity, such as when somebody replies to your post. However, when I click on them, I see an error message instead of the post.

# To Reproduce
1. Go to https://near.org/notifications
2. Click View for any notifications related to DevHub activity (e.g. Bo replied to your post)
3. See error: ```Source code for "near/widget/gigs-board.pages.Post" is not found```

# Expected behavior
Clicking on notification should take me to the post.

# Screenshots

## Notification Screen
![Screenshot 2023-11-22 at 11 14 15 AM](https://github.com/near/neardevhub-widgets/assets/110252255/d97531ab-6c24-4caa-9b99-45cf05289704)

## Error Message
![Screenshot 2023-11-22 at 11 14 34 AM](https://github.com/near/neardevhub-widgets/assets/110252255/7e705eac-401c-4dc8-87d0-a6bc6718537b)

# Other Details
**Desktop (please complete the following information):**
 - OS: OS
 - Browser: Safari
 - Version: Version 17.0 (19616.1.27.211.1)

**Comment by Tguntenaar** [comment]
`devgovgigs.near` is missing in the link so it wouldn't give an error if it was: `devgovgigs.near/widget/gigs-board.pages.Post` even better is to link the new page: `devhub.near/widget/app?page=post&id=${id}` however this is not controlled by this repository. @frol  @ailisp Who is in control of the notifcation service worker from near.org?

**Comment by elliotBraem** [comment]
@Megha-Dev-19 has opened PR with near discovery components repository: https://github.com/near/near-discovery-components/pull/544

**Comment by Megha-Dev-19** [comment]
@elliotBraem I just tested it and it works for mention. It redirects me to: 
https://near.org/devgovgigs.near/widget/gigs-board.pages.Post?id=2410
I m curious to know why "mention" works and why doesn't "reply" works.? Because in [near discover code](https://github.com/near/near-discovery-components/blob/develop/src/NearOrg/Notifications/Notification.jsx#L194) I see we have the same redirect URL for all

![Image](https://github.com/near/neardevhub-widgets/assets/100185149/c07947e5-f474-410b-a879-61c6381f6f69)

**Comment by elliotBraem** [comment]
Why is it redirecting to `https://near.org/devgovgigs.near/widget/gigs-board.pages.Post?id=2410`?

It should be `https://near.org/devhub.near/widget/app?page=post&id=2410`; I don't think these changes get merged in their side until Thursday -- they have a workflow for graduating changes on develop branch to main.

**Comment by Megha-Dev-19** [comment]
Yeah that’s what I meant, the changes are not yet merged and deployed. So this is the existing flow (with old code), why is it different for “reply” and “mention” notification types.?

**Comment by Megha-Dev-19** [comment]
@elliotBraem the [PR](https://github.com/near/near-discovery-components/pull/544) is merged

**Comment by elliotBraem** [comment]
Just verified this has been merged and this bug has been resolved. Closing.
